src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age !== undefined && <p className="Person__age">I am {age}</p>}
```

Comment:
```suggestion
      {age  && <p className="Person__age">I am {age}</p>}
```
Unnecessary comparison